databaseconnect.py

```diff
@@ -57,9 +57,8 @@ def setup_database():
 def add_to_database(classification, subject, root, verb, H):
     db = connection_to_database()
     cur = db.cursor()
-    cur = db.cursor(buffered=True)
+    cur = db.cursor(prepared=True)
```

Comment:
What is the reasoning behind this change?

Reply:
@vishakha-lall Since I am using prepared statements, this change is necessary as a rule.